backend/btrixcloud/crawls.py

```diff
@@ -68,11 +68,19 @@ async def init_index(self):
         await self.crawls.create_index(
             [("type", pymongo.HASHED), ("state", pymongo.DESCENDING)]
         )
+        await self.crawls.create_index(
+            [("type", pymongo.HASHED), ("fileSize", pymongo.DESCENDING)]
+        )
+        await self.crawls.create_index(
```

Comment:
I wonder if fileCount is actually useful metric to search for.. as it doesn't necessarily correspond to user controlled settings. I guess scale increases fileCount, but also long-running and larger crawler will have larger fileCounts..

Reply:
Hm yeah, good point, I'll remove it. We had had it in the crawls endpoint but maybe it's just noise.

Reply:
And done